data/movie/The Peanuts Movie (2015)/collectors-edition/disc01-summary.txt

```diff
@@ -0,0 +1,451 @@
+Duration: 0:10:24
+Chapters count: 1
+Size: 1.9 GB
+Segment count: 2
+Segment map: 2400,2398
+File name: We're in a Movie, Charlie Brown.mkv
+
+Name: You Never Grow Up Charlie Brown - Good 'Ol Charles Schulz and the Peanuts Gang
+Type: Extra
+Source file name: 00379.mpls
+Duration: 0:13:00
+Chapters count: 1
+Size: 2.4 GB
+Segment count: 2
+Segment map: 2401,2398
+File name: Good 'Ol Charles Schulz and the Peanuts Gang.mkv
+
+Name: Snoopy and Charlie Brown: The Peanuts Movie
+Type: MainMovie
+Year: 2015
+Chapters:
+- Chapter 1
+- Chapter 2
+- Chapter 3
+- Chapter 4
+- Chapter 5
+- Chapter 6
+- Chapter 7
+- Chapter 8
+- Chapter 9
+- Chapter 10
+- Chapter 11
+- Chapter 12
+- Chapter 13
+- Chapter 14
+- Chapter 15
+- Chapter 16
+- Chapter 17
+- Chapter 18
+- Chapter 19
+- Chapter 20
+- Chapter 21
+- Chapter 22
+- Chapter 23
+- Chapter 24
+Description: Includes alternate title card sequence
+Source file name: 00801.mpls
+Duration: 1:28:06
+Chapters count: 24
+Size: 24.4 GB
+Segment count: 3
+Segment map: 2586,2591,2589
+File name: Snoopy and Charlie Brown (2015) [1080p].mkv
+
+Name: The Peanuts Movie
+Type: MainMovie
+Year: 2015
+Chapters:
+- Chapter 0
+- Chapter 1
+Description: Omits studio opening sequences and title card
+Source file name: 00373.mpls
+Duration: 1:26:37
+Chapters count: 1
+Size: 24.0 GB
+Segment count: 2
+Segment map: 2591,2589
+File name: The Peanuts Movie (2015) without opening [1080p].mkv
+
+Name: Sneak Peek - Ice Age: Collision Course Cosmic Scrat-Tastrophe
+Type: Extra
+Source file name: 02600.m2ts
+Duration: 0:05:04
+Size: 758.4 MB
+Segment count: 1
+Segment map: 2600
+File name: Ice Age Collision Course Cosmic Scrat-Tastrophe.mkv
+
+Name: Sneak Peek - Alvin and the Chipmunks: The Road Chip
+Type: Trailer
+Source file name: 00386.mpls
+Duration: 0:02:25
+Chapters count: 1
+Size: 338.4 MB
+Segment count: 2
+Segment map: 2563,2599
+File name: The Road Chip trailer.mkv
+
+Name: Sneak Peek - Kung Fu Panda 3
+Type: Trailer
+Source file name: 00384.mpls
+Duration: 0:02:26
+Chapters count: 1
+Size: 340.2 MB
+Segment count: 2
+Segment map: 2562,2598
+File name: Kung Fu Panda 3 trailer.mkv
+
+Name: Sneak Peek - Home
+Type: Trailer
+Source file name: 02569.m2ts
+Duration: 0:00:17
+Size: 38.8 MB
+Segment count: 1
+Segment map: 2569
+File name: Home trailer.mkv
+
+Name: Sneak Peek - Strawberry Shortcake: Berry Tales
+Type: Trailer
+Source file name: 02571.m2ts
+Duration: 0:00:17
+Size: 38.9 MB
+Segment count: 1
+Segment map: 2571
+File name: Berry Tales trailer.mkv
+
+Name: Sneak Peek - The Sound of Music 50th Anniversary
+Type: Trailer
+Source file name: 00348.mpls
+Duration: 0:00:54
+Chapters count: 1
+Size: 124.2 MB
+Segment count: 2
+Segment map: 2570,2594
+File name: The Sound of Music 50th Anniversary trailer.mkv
+
+Name: Japanese anti-piracy warning
+Type: Extra
+Source file name: 00123.m2ts
+Duration: 0:00:20
+Size: 50.0 MB
+Segment count: 1
+Segment map: 123
+File name: Japanese anti-piracy warning.mkv
```

Comment:
I am interested to hear why you want to include this. I definitely don't think this is an extra. Do you think anyone wants to identify and rip piracy warnings?

Reply:
lol, I just thought it was novel, forgetting that it would be quite routine in Japan. You are right that probably no one wants to rip this. Will remove it in the morning